DNA_RNA.py

```diff
@@ -0,0 +1,69 @@
+import string
+
+__nucleic_acid_seq_DNA__ = set('ACGT')
```

Comment:
```suggestion
__nucleic_acid_seq_DNA__ = set('ACGTacgt')
```
it doesn't work with lowercase letters